cannon/cannon.py

```diff
@@ -0,0 +1,190 @@
+# Author: Sathish Kumar E
+
```

Comment:
Please name the file with capital: `Cannon`.

Great name, I was thinking the same name btw:)